app/javascript/components/patient/laboratory/Laboratory.js

```diff
@@ -111,10 +114,18 @@ class Laboratory extends React.Component {
             cancel={this.toggleLabModal}
             editMode={!!this.props.lab.id}
             loading={this.state.loading}
+            onlyPosLab={this.props.onlyPosLab}
+            isolation={this.props.patient.isolation}
           />
         )}
         {this.state.showDeleteModal && (
-          <DeleteDialog type={'Lab Result'} delete={this.handleDeleteSubmit} toggle={this.toggleDeleteModal} onChange={this.handleChange} />
+          <DeleteDialog
+            type={'Lab Result'}
+            delete={this.handleDeleteSubmit}
+            toggle={this.toggleDeleteModal}
+            onChange={this.handleChange}
+            promptSymptomOnset={this.props.patient.isolation && !this.props.patient.symptom_onset && this.props.onlyPosLab}
```

Comment:
I don't love this prop name, replacing show with prompt would be more consistent

Reply:
did you mean to suggest replacing `promptSymptomOnset` with `showSymptomOnset`?

Reply:
changed it to `showSymptomOnsetInput` for now: https://github.com/SaraAlert/SaraAlert/pull/861/commits/ea0697f83c29320b8930fb7a5dcf63015ea342fb#diff-431caf4d60fa1325346adae1e9dd1b818a55ea030365a593cdf8ebcaaab58dbdR128